Dynavity/Dynavity/view-model/HtmlExporter.swift

```diff
@@ -0,0 +1,5 @@
+import Combine
+
+protocol HtmlExporter {
```

Comment:
The current protocol name doesn't tell us that it is used to render markup text. Perhaps this should be named `MarkupToHtmlExporter` instead?

Then the existing `MarkupToHtmlExporter` could be renamed to specify the implementation used (pandoc).

Reply:
Good point. thanks!